docs/source/autograd.rst

```diff
@@ -11,6 +11,33 @@ Automatic differentiation package - torch.autograd
 
 .. autofunction:: grad
 
+.. _functional-api:
+
+Functional higher level API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section contains the higher level API for the autograd that build on the basic API above.
```

Comment:
nit: "that build on the basic api above" -> "that builds on the basic api above"